### PR TITLE
Refactor existing Tests and Reports -pages to current architecture

### PR DESCRIPTION
**Pull request by mikkoaf**
In this changes have been made to the Tests and Reports -pages in order to make future changes easier. The pages are now more similar to how the Cruft remover and Shadows -pages work. With these changes it's easier to redesign the plugin menu layout as described in #98 and #142.

### CODE REVIEW

File: lib/tests-page.php [hunk 1]
```diff
@@ -1,42 +1,59 @@
 <?php
 // Deny direct access to this file
 if ( ! defined('ABSPATH') ) {
-  die('Access denied!');
+    die('Access denied!');
```
Comment (l3ku) [inline lib/tests-page.php]: Please revert this change, as the indentation was already correct with two spaces.

**Comment by mikkoaf** [comment]
In the current iteration the "Show more" link behaves better and the unnecessary modification on line 4 is now reverted.
I have planned on improving upon how the postboxes are presented to the user at a later time, as with this PR the intention is to make it easier to rearrange the menu.
